autoload/clever_f.vim

```diff
@@ -96,6 +101,43 @@ function! s:is_timedout() abort
     return elapsed_ms > g:clever_f_timeout_ms
 endfunction
 
+function! s:mark_direct(forward, count) abort
+    let line = getline('.')
+    let c_start = col('.')
+    let c_end = col('.')-2
+    let l = line('.')
+
+    if (a:forward && c_start+1 > len(line)-1) ||
+        \(!a:forward && c_end+1 < 0)
+        return []
+    endif
+
+    if g:clever_f_ignore_case
+        let line = tolower(line)
+    endif
+
+    let r = a:forward ? range(c_start, len(line)-1)
+        \             : range(c_end, 0, -1)
+    let d = {}
+    let ms = []
```

Comment:
Please avoid short name for relatively long-live variables such as `r`, `d`, `ms`. They are not suitable for maintenance. For example, I'm not understanding what `d` stands for.

Reply:
renamed